mgcpy/independence_tests/unit_tests/mgc/local_correlation_test.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import pytest
-from mgcpy.independence_tests.mgc.local_correlation import local_correlations
+from mgcpy.independence_tests.utils.local_correlation import local_correlations
```

Comment:
local_correlations and threshold_smooth are not generic functions that can be in the utils, they are MGC specific, do not confuse them by moving out of mgc module.